html-css/week1/Vadim/OrderProject/files/css/styles.css

```
@@ -0,0 +1,405 @@
+body {
+  margin: 0 auto;
+  font-family: Arial;
+}
+
+body a {
+  cursor: pointer;
+}
+
+/* --- Header --- */
+.order {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  background: linear-gradient(92.74deg, #FD7B46 1.15%, #FE4E9B 98.48%);
+  padding: 2%;
+}
+
+.order__summary {
+  align-self: center;
```

Use the following if you delete the flex properties from the parent element.
```suggestion
  text-align: center;
```

---

makes sense!